tests/automation_framework/tests/work_order_tests/test_getresult.py

```diff
@@ -0,0 +1,256 @@
+    @pytest.mark.sdk
+    def test_workordergetresult_workorderid_alphabetsonly(self):
+        request_file = os.path.join(
+            constants.work_order_input_file,
+            "workordergetresult_workorderid_alphabetsonly.json")
+
+        err_cd = \
+            self.test_obj.setup_and_build_request_wo_getresult(
+                read_json(request_file))
+
+        submit_response = submit_request(
+            self.test_obj.uri_client,
+            self.test_obj.build_request_output['request_obj'],
+            constants.wo_submit_output_json_file_name,
+            read_json(request_file))
+        logger.info("submit_response: \n%s\n", submit_response)
+
+        assert (
+                check_negative_test_responses(
+                    submit_response,
+                    "Invalid work order Id")
+                is TestStep.SUCCESS.value)
+        logger.info('\t\t!!! Test completed !!!\n\n')
+
+    @pytest.mark.workordergetresult
+    @pytest.mark.test_workordergetresult_workorderid_withoutquotes
+    @pytest.mark.sdk
+    def test_workordergetresult_workorderid_withoutquotes(self):
+        request_file = os.path.join(
+            constants.work_order_input_file,
+            "workordergetresult_workorderid_withoutquotes.json")
+
+        err_cd = \
+            self.test_obj.setup_and_build_request_wo_getresult(
+                read_json(request_file))
+
+        submit_response = submit_request(
+            self.test_obj.uri_client,
+            self.test_obj.build_request_output['request_obj'],
+            constants.wo_submit_output_json_file_name,
+            read_json(request_file))
+        logger.info("submit_response: \n%s\n", submit_response)
+
+        assert (
+                check_negative_test_responses(
+                    submit_response,
+                    "Invalid params")
+                is TestStep.SUCCESS.value)
+
+        logger.info('\t\t!!! Test completed !!!\n\n')
+
+    @pytest.mark.workordergetresult
+    @pytest.mark.test_workordergetresult_emptyparameter
+    def test_workordergetresult_emptyparameter(self):
+        request_file = os.path.join(
+            constants.work_order_input_file,
+            "workordergetresult_emptyparameter.json")
+
+        err_cd = \
+            self.test_obj.setup_and_build_request_wo_getresult(
+                read_json(request_file))
+
+        submit_response = submit_request(
+            self.test_obj.uri_client,
+            self.test_obj.build_request_output['request_obj'],
+            constants.wo_submit_output_json_file_name,
+            read_json(request_file))
+        logger.info("submit_response: \n%s\n", submit_response)
+
+        assert (validate_response_code(submit_response, 2)
+                is TestStep.SUCCESS.value)
+
+        logger.info('\t\t!!! Test completed !!!\n\n')
+
+
+    @pytest.mark.workordergetresult
+    @pytest.mark.test_workordergetresult_unknownparameter
```

Comment:
Does this need an SDK marker?

Reply:
This will not work with SDK.  hence removed the marker. kept the test ready to work with listener.